src/Shared/Http2/Hpack/HPackDecodingException.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0.
```

Comment:
Add an exclusion to the license.txt file for this folder using a different license.

Reply:
Do we have an example for that? I don't see any other exclusions.

Reply:
Actually I think you are good, David added the exclusion in his last commit: https://github.com/aspnet/AspNetCore/commit/bd65275148abc9b07a3b59797a88d485341152bf